### PR TITLE
177090632 location model

### PR DESCRIPTION
**Pull request by abishekc**

### CODE REVIEW

**Review by chrononyan — APPROVED** [review]

**Comment by klutzx92** [comment]
**What is the goal of this PR?**
_Finishing location model and testing it. They added some fields to the location model and added RSpec tests._

**Were you able to view screenshots, story cards, and or test the app?**
_Couldn't find any screenshots, story cards, or links to the deployed app._

**Is the PR in a shape that makes it easy to review?**
_Couldn't find description for PR. Review had to be based on the commit messages and files._ 

**What are the status of the tests?**
Are the tests included all passing? Are the tests comprehensive? Are there any features that need to be tested? Are there any tests that are redundant?
_Yes they all pass. The tests seem comprehensive enough. All the features are tested._ 

**What is the style of the code like?**
Are there any code linters that were run automatically?
What is their status?
If there are no linters, you get to play the human code linter.

_Doesn't seem like there were code linters. One file not ending with line break. Indentation is consistent. Variable names look pretty good._ 

**Is there anything missing from this PR that should be included?**
_The description was missing as well as any screenshots/links to deployed app. Also person approving the PR did not leave any comments such as "LGTM". But commit messages were descriptive and things were kept simple._

**Feature Design: Are there any parts of this feature that are cause for concern?**
There might be nothing. Could this feature be abused by users? Does the code look safe in how it treats data? Are there any potential bugs that should be guarded against?
_The feature design looks good. Its a pretty simple PR that doesn't have too much complexity._

---

### Overall Assessment

**What does this PR do well?**
_Keeps feature simple, comprehensive testing included, commit messages are descriptive, and branch is closed immediately after merging._

**What is something that could be improved with this PR?**
_Add a description for the PR, add approval comments, and linters/style/linebreak._

File: app/models/location.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Location < ApplicationRecord
+	validates_presence_of :name
+	validates_presence_of :description
+	validates_presence_of :longitude
+	validates_presence_of :latitude
+end
```
Comment (klutzx92) [inline app/models/location.rb:6]: LGTM

File: spec/models/location_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      subject.latitude = nil
+      expect(subject).to_not be_valid
+    end
+
+    it "is not valid without a longitude" do
+      subject.longitude = nil
+      expect(subject).to_not be_valid
+    end
+  end
+end
```
Comment (klutzx92) [inline spec/models/location_spec.rb:33]: Missing line break

File: spec/models/location_spec.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    it "is not valid without a latitude" do
+      subject.latitude = nil
+      expect(subject).to_not be_valid
+    end
+
+    it "is not valid without a longitude" do
+      subject.longitude = nil
+      expect(subject).to_not be_valid
+    end
```
Comment (klutzx92) [inline spec/models/location_spec.rb:31]: Excellent test coverage!

**Pull request merged**